test/_features/example_app/save_and_return.feature

```diff
@@ -45,4 +45,61 @@ Feature: Save and return
     Then I should be on the 'name' page showing 'What is your name?'
     Then I fill 'name' with 'Jane Doe'
     Then I click the 'Save and exit' button
-    Then I should be on the 'save-and-return' page showing 'Save and return feature'
\ No newline at end of file
+    Then I should be on the 'save-and-return' page showing 'Save and return feature'
+
+  @delete_form
+  Scenario: Creating and deleting a form
+    Given I start the 'feature' application journey
+    Then I should be on the 'save-and-return' page showing 'Save and return feature'
+    Then I click the 'Start a form' button
+    Then I should be on the 'start' page showing 'Enter your email address'
+    Then I fill 'saveEmail' with 'testdelete@email.com'
+    Then I click the 'Save and continue' button
+    Then I should be on the 'forms' page showing 'Create a new form'
+    Then I continue to the next step
+    Then I should be on the 'reference' page showing 'Give your form a reference'
+    Then I fill 'reference' with '1234'
+    Then I click the 'Save and continue' button
+    Then I should be on the 'name' page showing 'What is your name?'
+    Then I fill 'name' with 'Jane Doe'
+    Then I click the 'Save and exit' button
+    Then I click the 'Start a form' button
+    Then I should be on the 'start' page showing 'Enter your email address'
+    Then I fill 'saveEmail' with 'testdelete@email.com'
+    Then I click the 'Save and continue' button
+    Then I should be on the 'forms' page showing 'Create a new form'
+    Then I click the 'Delete form' button
+    Then I click the 'Delete this form' button
+    Then I should be on the 'forms' page showing 'Create a new form'
+
+  @delete_form @check_after_deletion
```

Comment:
Don't think you need to create a new test for this section. Could just add to the above to make it more concise/ When they get redirected to the 'form' could you not just check that there is the following 'You do not currently have any draft forms' on the page.

Reply:
Don't need to add the extra step definition then.

Reply:
merged the tests so they're more concise and removed extra step definition

Reply:
The 'Then I navigate...' step definition is still in the step_definitions file.